korlibs-logger/src@linux/korlibs/logger/Logger.linux.kt

```diff
@@ -0,0 +1,28 @@
+@file:OptIn(ExperimentalForeignApi::class)
+
+package korlibs.logger
+
+import kotlinx.cinterop.*
+
+actual object Console : BaseConsole()
+
+actual object DefaultLogOutput : Logger.Output {
+    override fun output(logger: Logger, level: Logger.Level, msg: Any?) = Logger.ConsoleLogOutput.output(logger, level, msg)
+}
+
+internal actual val miniEnvironmentVariables: Map<String, String> by lazy {
+    getEnvs()
+}
+
+private fun getEnvs(): Map<String, String> {
+    val out = LinkedHashMap<String, String>()
+    val env = platform.posix.__environ
```

Comment:
We should avoid so short variable names. It really makes the code hard to read.

Reply:
We could control env local variable, but not __environ since that comes from the operating system